shoop/core/fields/__init__.py

```diff
@@ -60,23 +62,40 @@ def __init__(self, **kwargs):
         super(CurrencyField, self).__init__(**kwargs)
 
 
-class MoneyValueField(models.DecimalField):
+class FormattedDecimalField(models.DecimalField):
+    """
+    DecimalField subclass that truncates zeroes to avoid displaying in
+    scientific notation.
+    """
+    def _remove_exponent(self, val):
+        context = decimal.Context(prec=self.max_digits)
+        if val == val.to_integral():
+            return val.quantize(decimal.Decimal(1), context=context)
+        else:
+            return val.normalize(context)
+
+    def value_from_object(self, obj):
+        val = super(FormattedDecimalField, self).value_from_object(obj)
+        if isinstance(val, decimal.Decimal):
+            return self._remove_exponent(val)
```

Comment:
Maybe you could just use the new formatting operator (from py 2.7 / py 3), i.e. `'{:f}'.format(val)` or directly using the `format` function: `format(val, 'f')`.
```
>>> format(1/decimal.Decimal(10**100), 'f')
'0.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001'
```

It will return a string,  so I'm not sure which function of the Field you should override, but I'm sure you'll figure that out. :)